### DISCUSSION TITLE
Running 3.12.0 in Kubernetes

### DISCUSSION THREAD

**Discussion by paulbrai** [Netflix/conductor#3274] [General]
I am trying to run Netflix conductor in Kubernetes based on the containers built with docker compose.  Although it starts and the logs for both the UI and server seem devoid of errors, it doesn't actually work.  All I see in the UI is the a spinning wheel that says "Loading Options" that never actually load.
I am presuming I need to set the right options in config.properties and ensure it loads, but thus far it seems to be a form of mysticism to actually know what to put in config.properties.  This is what I have so far:
```
    conductor.grpc-server.enabled=false
    conductor.db.type=redis_standalone
    workflow.dynomite.cluster.name=redis
    conductor.redis.hosts=redis.conductor.svc.cluster.local:6379:1
    workflow.namespace.prefix=conductor
    workflow.namespace.queue.prefix=conductor_queues
    queues.dynomite.nonQuorum.port=6379
    workflow.dynomite.cluster.hosts.useSSL=false
    conductor.redis.availabilityZone=us-west-2c
    conductor.indexing.enabled=true
    conductor.elasticsearch.url=http://elastic-conductor.elastic.svc.cluster.local:9200
    conductor.elasticsearch.indexReplicasCount=0
    conductor.elasticsearch.indexName=conductor
    conductor.elasticsearch.clusterHealthColor=yellow
    loadSample=true
```
Where there is the simplest possible single node redis deployment and elastic 6.8.22 deployment running in the appropriate namespaces.
I'd love to be troubleshooting error messages but since it seems basically impossible to get debug messaging I have to ask here.

**Comment by apanicker-nflx** [comment]
@v1r3n Could you please help out with the Kubernetes setup that you use?
